src/components/board/component.test.jsx

```diff
@@ -4,11 +4,52 @@ import { mount } from 'enzyme';
 import Board from './component';
 
 describe('<Board/>', () => {
-  it('onClick event works on buttons', () => {
-    const onClick = jest.fn();
-    const squares = Array(9).fill('');
-    const wrapper = mount(<Board squares={squares} onClick={onClick} />);
-    wrapper.find('button.square').first().simulate('click');
+  const onClick = jest.fn();
+  const squares = Array(9).fill('');
+  const wrapper = mount(<Board squares={squares} onClick={onClick} />);
+
+  it('onClick event works on 1st button', () => {
+    wrapper.find('button.square').at(0).simulate('click');
+    expect(onClick).toBeCalledWith(0);
+  });
+
+  it('onClick event works on 2nd button', () => {
+    wrapper.find('button.square').at(1).simulate('click');
+    expect(onClick).toBeCalledWith(0);
```

Comment:
Shouldn't `onClick` be called with `1`? Same for tests in this file

Reply:
Yes, I should actually